service/src/main/resources/application-control-plane.yml

```
@@ -11,6 +11,5 @@ twds:
 
   data-import:
     batch-write-record-sink: "rawls"
-    rawls-bucket:
-      name: ${SERVICE_GOOGLE_BUCKET:unknown}
-      projectId: ${SERVICE_GOOGLE_PROJECT:unknown}
+    project-id: ${SERVICE_GOOGLE_PROJECT:unknown}
+    rawls-bucket-name: ${SERVICE_GOOGLE_BUCKET:unknown}
```

I think better to set these as empty instead of the magic string "unknown":
```suggestion
    project-id: ${SERVICE_GOOGLE_PROJECT}
    rawls-bucket-name: ${SERVICE_GOOGLE_BUCKET}
```